product_docs/docs/migration_toolkit/55/07_invoking_mtk/08_mtk_command_options.mdx

```diff
@@ -315,11 +315,19 @@ MySQL users note: By default, the MySQL JDBC driver fetches all of the rows in a
 
 `-filterProp <file_name>`
 
- `<file_name>` specifies the name of a file that contains constraints in key=value pairs. Each record read from the database is evaluated against the constraints. Those that satisfy the constraints are migrated. The left side of the pair lists a table name. The table name must not be schema qualified. The right side specifies a condition that must be true for each row migrated. For example, including the following constraints in the property file
+ `<file_name>` specifies the name of a file that contains constraints in key=value pairs. Each record read from the database is evaluated against the constraints. Those that satisfy the constraints are migrated. 
+ 
+ The left side of the pair lists a table name: 
+ - The table name must not be schema qualified. 
+ - Oracle table names are not case sensitive, but Oracle defaults to uppercase. Postgres, on the other hand defaults to lowercase. When migrating Oracle data, make sure that the case you are using matches the case in Oracle; otherwise, Migration Toolkit can't match it and ignores the constraint. 
```

Comment:
```suggestion
 - Oracle table names are not case sensitive, but Oracle defaults to uppercase. Postgres, on the other hand, defaults to lowercase. When migrating Oracle data, make sure that the case you are using matches the case in Oracle; otherwise, Migration Toolkit can't match it and ignores the constraint. 
```